test/com/google/javascript/jscomp/PolymerBehaviorExtractorTest.java

```diff
@@ -28,8 +27,6 @@
  */
 public class PolymerBehaviorExtractorTest extends CompilerTypeTestCase {
 
-  private static final Joiner LINE_JOINER = Joiner.on('\n');
```

Comment:
Ah, I missed that one. They should yes. But then let's get rid of this one because it's confusing to have both: https://github.com/google/closure-compiler/blob/b0b676c18657f6d8590c79de47c625e170ebae3d/test/com/google/javascript/jscomp/CompilerTypeTestCase.java#L69